src/config/interfaces/Config/IURLConfig.ts

```diff
@@ -93,6 +94,11 @@ interface IURLConfig {
    * Whether or not to force a version
    */
   forceVersion?: boolean;
+
+  /**
+   * Additional query params to be added to the URL
+   */
+  queryParams?: Record<string, string | number>
```

Comment:
I'm thinking if the name should rather be aligned with web api (URLSearchParams) so that would mean `searchParams`. We can have a discussion about it.